Rakefile

```diff
@@ -37,10 +48,25 @@ A quick acceptance test, named because it has no pre-suites to run
 
 end
 
+namespace :generate do
+  desc "Generate test fixtures."
+  task :fixtures do
+    $LOAD_PATH.unshift(
+      File.join(Dir.pwd, 'lib'),
+      File.join(Dir.pwd, 'test')
+    )
+    require 'beaker-hostgenerator'
+    require 'util/generator_helpers'
+
+    fixgen = FixtureGenerator.new
+    fixgen.generate
+  end
+end
+
 # namespace-named default tasks.
 # these are the default tasks invoked when only the namespace is referenced.
 # they're needed because `task :default` in those blocks doesn't work as expected.
-task 'test:spec' => 'test:spec:run'
+task 'test:spec' => ['test:spec:run', 'test:spec:minitest']
```

Comment:
No, see the documentation at:
https://github.com/puppetlabs/beaker-hostgenerator/blob/qeng-3370/README.md#generated-fixtures

The fixture generator is not meant to be run every time the tests are run--that's why the fixtures are committed to the repo, to "snapshot" the current behavior of beaker-hostgenerator in a way that helps prevent regressions as we refactor or repurpose parts of the code.

Anyway, we generated fixtures before every time we run the specs that use those fixtures that spec would always pass unless we do something complicated to generate the fixtures for some older commit of the repo then test using them against the HEAD of the PR branch (which would assume a certain CI setup).